Framework/Json/inc/MantidJson/Json.h

```diff
@@ -0,0 +1,22 @@
+// Mantid Repository : https://github.com/mantidproject/mantid
+//
+// Copyright &copy; 2021 ISIS Rutherford Appleton Laboratory UKRI,
+//   NScD Oak Ridge National Laboratory, European Spallation Source,
+//   Institut Laue - Langevin & CSNS, Institute of High Energy Physics, CAS
+// SPDX - License - Identifier: GPL - 3.0 +
+#pragma once
+
+#include "MantidJson/DllConfig.h"
+
+#include <json/reader.h>
+#include <json/value.h>
+#include <json/writer.h>
+#include <string>
+
+namespace Mantid::JsonHelpers {
```

Comment:
For consistency with other libraries and what developers would expect can we name this `namespace Mantid::Json`?

Reply:
As explained earlier, I cannot, due to the fact it trys to resolve Mantid::Json for Json::Value's in other libraries and this libraries.

Reply:
My apologies - it slipped my mind that this was the case. How about if we added a `using Value = ::Json::Value;` inside the `namespace Mantid::Json {` namespace ?

Reply:
I think this wouldn't solve the issues in all the other files that include both JsonHelpers and Json::Value or any other JsonCpp class of which most files have both or none.